src/structure/spatial.jl

```diff
@@ -187,6 +187,23 @@ function omegaangle(res::AbstractResidue, res_prev::AbstractResidue)
 end
 
 """
+Calculate the omega angle in radians for a residue in a `Chain`. Arguments are
```

Comment:
I think the docstrings can go as an extension to the docstrings in the above function. So change the above docstrings to be something like "Arguments can either be a residue and the previous residue or a chain and the position as an integer".